en/backup-to-aws-s3-using-br.md

```diff
@@ -398,6 +404,8 @@ More `Backup` CR parameter description:
 
 In the examples above, some parameters in `.spec.br` can be ignored, such as `logLevel`, `statusAddr`, `concurrency`, `rateLimit`, `checksum`, `timeAgo`, and `sendCredToTikv`.
 
+If you want to back up incrementally, you only need to specify the last backup timestamp `--lastbackupts` in `spec.br.options`. You can refer to [Use BR to Back up and Restore Data](https://docs.pingcap.com/tidb/stable/backup-and-restore-tool#back-up-incremental-data) for the limitations of incremental backup.
```

Comment:
```suggestion
Since TiDB Operator v1.1.6, if you want to back up incrementally, you only need to specify the last backup timestamp `--lastbackupts` in `spec.br.options`. You can refer to [Use BR to Back up and Restore Data](https://docs.pingcap.com/tidb/stable/backup-and-restore-tool#back-up-incremental-data) for the limitations of incremental backup.
```

Reply:
Fixed in a new commit.